spec/Menu_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'Menu'
```

Comment:
Style/FileName: Use snake_case for source file names. (https://github.com/bbatsov/ruby-style-guide#snake-case-files)